cmd/zoekt-mirror-github/main.go

```diff
@@ -68,6 +69,7 @@ func main() {
 	flag.Var(&topics, "topic", "only clone repos whose have one of given topics. You can add multiple topics by setting this more than once.")
 	excludeTopics := topicsFlag{}
 	flag.Var(&excludeTopics, "exclude_topic", "don't clone repos whose have one of given topics. You can add multiple topics by setting this more than once.")
+	onlyActive := flag.Bool("only-active", false, "mirror only active projects")
```

Comment:
Can we change the description to mention "archived" instead of "active" because in the context of GitHub is not clear what "active" means.

Maybe "no_archived" makes more sense? The filter function would become more readable, too.

```
if *noArchived && *repo.Archived {
    continue
}
filteredRepos = append(filteredRepos, repo)
```